net-libs/configurable-http-proxy/configurable-http-proxy-9999.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+DESCRIPTION="A simple wrapper around node-http-proxy that adds a REST API for updating the routing table"
+HOMEPAGE="http://jupyter.org"
+
+if [ ${PV} == "9999" ] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/jupyter/${PN}.git git://github.com/jupyter/${PN}.git"
+else
+	SRC_URI="https://github.com/jupyter/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+
+RDEPEND="
+	>=net-libs/nodejs-0.8.28[npm]
+	"
+DEPEND="${RDEPEND}"
+
+src_install() {
+	# npm_src_install() from https://github.com/neurogeek/gentoo-overlay/blob/master/eclass/npm.ecla
```

Comment:
@jlec, I discussed this by email with @neurogeek some time ago. One of the main reasons why the npm ebuilds and the eclass didn't made it to the tree is that npm often uses local installations instead of global installations and therefore installs package duplicates. Further, versioning of npm packages isn't really compatible with ebuild versioning. Thus as long as there are no a bundle of packages requiring a npm.eclass, I would try to avoid the effort.